cookbooks/muparser_temperature_example/muparser-temperature-example.prm

```diff
@@ -63,6 +63,9 @@ end
 # Refine the upper 150 km of the mesh so lithosphere structure is resolved.
 # Function expression asks if the depth is less than the lithosphere depth and
 # then refinement level = 5, else  refinement level = 0
+# Note that this refinement level 0 means the refinement level below 150 km 
+# depth is the same as initial global refinement level (3), rather than no 
+# mesh refinement 
```

Comment:
Just to be safe, can you check that this in fact the case in this specific cookbook? I know you have tested this in other examples.

Reply:
Tested the case and corrected the words